src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --color-hover: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.img {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  height: 60px;
+}
```

Comment:
You've fixed the height of the `.nav` class. This can lead to issues with content overflow. It's better to let the content determine the size of the container unless there's a specific design requirement to fix the size.